### ISSUE TITLE
ERROR: Exception on /search/

### ISSUE THREAD

**Issue by TheoBr** [metpetdb/interface_v2#19]
Currently deployed version is broken. Found this in the logs

```
[Tue Dec 13 05:24:00.519151 2016] [:error] [pid 2374] ERROR:interface_v2:Exception on /search/ [GET]
[Tue Dec 13 05:24:00.519206 2016] [:error] [pid 2374] Traceback (most recent call last):
[Tue Dec 13 05:24:00.519215 2016] [:error] [pid 2374]   File "/home/metpet/.virtualenvs/metpetdb/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
[Tue Dec 13 05:24:00.519224 2016] [:error] [pid 2374]     response = self.full_dispatch_request()
[Tue Dec 13 05:24:00.519232 2016] [:error] [pid 2374]   File "/home/metpet/.virtualenvs/metpetdb/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
[Tue Dec 13 05:24:00.519239 2016] [:error] [pid 2374]     rv = self.handle_user_exception(e)
[Tue Dec 13 05:24:00.519247 2016] [:error] [pid 2374]   File "/home/metpet/.virtualenvs/metpetdb/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
[Tue Dec 13 05:24:00.519255 2016] [:error] [pid 2374]     reraise(exc_type, exc_value, tb)
[Tue Dec 13 05:24:00.519263 2016] [:error] [pid 2374]   File "/home/metpet/.virtualenvs/metpetdb/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
[Tue Dec 13 05:24:00.519271 2016] [:error] [pid 2374]     rv = self.dispatch_request()
[Tue Dec 13 05:24:00.519279 2016] [:error] [pid 2374]   File "/home/metpet/.virtualenvs/metpetdb/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
[Tue Dec 13 05:24:00.519288 2016] [:error] [pid 2374]     return self.view_functions[rule.endpoint](**req.view_args)
[Tue Dec 13 05:24:00.519296 2016] [:error] [pid 2374]   File "/home/metpet/metpetdb/interface_v2/__init__.py", line 51, in search
[Tue Dec 13 05:24:00.519304 2016] [:error] [pid 2374]     regions = get(env("API_HOST")+"regions/", params = {"fields": "name", "page_size": 2000, "format": "json"}).json()["results"]
[Tue Dec 13 05:24:00.519313 2016] [:error] [pid 2374] TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

Bottom 4-ish lines are the important bit

**Comment by TheoBr** [comment]
This is caused by the dotenv being read in two different places (both different file paths and at different places in the app.py)

https://github.com/metpetdb/interface_v2/blob/master/app.py#L24

https://github.com/metpetdb/interface_v2/blob/master/app.py#L739

**Comment by kienankb** [comment]
Fixed by #22.
